code/src/Colorpicker.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+
+export const Colorpicker = (props) => {
+  const [color, setColor] = useState('#e66465');
+
+  const handleChange = (event) => {
+    setColor(event.target.value);
+  }
+
+  const clickNextButton = () => {
+    props.onChange(color);
+  }
+
+  return (
+    <div className="question-container">
+      <label htmlFor="color" className="question-label">{props.text}  </label>
+      <input
+        type="color"
+        id="color"
+        name="favcolor"
+        value={color}
+        onChange={handleChange} />
+
+      <button
+        className="next-button"
+        type="button"
+        onClick={() => clickNextButton()}>
```

Comment:
Since you are not passing any arguments inside `clickNextButton()` function, we can simplify this line of code :

```
onClick={clickNextButton}>
```

What's the difference? With your approach, you are creating new function each time you click on button.
With my approach however we are creating function only one time, so we are more efficient